models/public/drn-d-38/README.md

```diff
@@ -0,0 +1,120 @@
+# drn-d-38
+
+## Use Case and High-Level Description
+
+The `drn-d-38` model is a one of the Dilated Residual Networks (DRN) models for semantic segmentation task. DRN models dilate ResNet models, DRN-C version additionally removes residual connections from some of the added blocks and DRN-D version is a simplified version of DRN-C.
+
+This model pre-trained on [Cityscapes](https://www.cityscapes-dataset.com) dataset for 19 object classes, listed in `<omz_dir>/data/dataset_classes/cityscapes_19cl.txt` file. See Cityscapes classes [definition](https://www.cityscapes-dataset.com/dataset-overview) for more details.
```

Comment:
as I can see <omz_dir>/data/dataset_classes/cityscapes_19cl.txt contains 20 classes (extra class for background), this model does not contains this class. I understand that it should work with current labels (more is better than less), but logically it may confuse

Reply:
New file with classes has been added, model description has been updated